src/html/extractors/factories/elementAttribute.ts

```diff
@@ -3,12 +3,20 @@ import { HtmlUtils } from '../../utils';
 import { elementExtractor } from './element';
 import { Validate } from '../../../utils/validate';
 import { IHtmlExtractorOptions, validateOptions } from '../common';
+import { IContentOptions, getContentOptions, validateContentOptions } from '../../../utils/content';
 
 export function elementAttributeExtractor(selector: string, textAttribute: string, options: IHtmlExtractorOptions = {}): IHtmlExtractorFunction {
     Validate.required.nonEmptyString({selector, textAttribute});
     validateOptions(options);
+    validateContentOptions(options);
+
+    let contentOptions: IContentOptions = getContentOptions(options, {
+        trimWhiteSpace: true,
+        preserveIndentation: false,
```

Comment:
In order to avoid breaking changes I wonder if we should set this to `trimWhiteSpace: false` and `preserveIndentation: true`. What do you think?

(the same would apply to the defaults in `element.ts` of course)

Reply:
That's another good point, we should change the defaults accordingly!